catalog-spec/catalog-spec.md

```diff
@@ -30,19 +31,17 @@ incorporated.
 
 ## Catalog Definitions
 
-
 There are two required element types of a Catalog: Catalog and Item. A STAC Catalog
 points to [STAC Items](../item-spec/), or to other STAC catalogs. It provides a simple
-linking structure that can be used recursively so that many `Items` can be included in 
+linking structure that can be used recursively so that many Items can be included in 
```

Comment:
This isn't really true anymore. 
A Catalog is of course required, but if Items are required then Collections are required since Items must now point to a Collection. 
But we also talked about being able to create a Catalog containing a Collection that has no Items. So I think that Catalog and Collections are the two required elements, and Item is in fact optional.

Reply:
I think that's not true. Items don't require a Collection. Only Collections require Items to link back. But you could completely leave out collections and make a catalog just with only Catalogs and Items. See the discussion in https://github.com/radiantearth/stac-spec/pull/262#discussion_r223053198

It all depends on what you are using:
* Catalogs require Items.
* Collections don't require Items, but require Items to link back.
* Items don't require anything, not even a Catalog.